frontend/src/Views/Export/ExportPage.tsx

```diff
@@ -6,6 +6,9 @@
 
 import React from 'react';
 
-const ExportPage: React.FC = () => <h2>Hello World!</h2>;
+const ExportPage: React.FC = () => {
+  localStorage.clear();
```

Comment:
Will be better to be once the export to nmp button is clicked, clear localstorage.

Reply:
But that would be on an export page ticket